meerk40t/core/node/rootnode.py

```diff
@@ -111,6 +111,27 @@ def notify_modified(self, node=None, **kwargs):
             if hasattr(listen, "modified"):
                 listen.modified(node, **kwargs)
 
+    def notify_translated(self, node=None, dx=0, dy=0, **kwargs):
```

Comment:
This trick might work also work for uniform scaling. To know that the tree doesn't need new pictures.